src/python_testing/matter_testing_support.py

```diff
@@ -707,6 +708,16 @@ def check_pics(self, pics_key: str) -> bool:
         pics_key = pics_key.strip().upper()
         return pics_key in picsd and picsd[pics_key]
 
+    def openCommissioningWindow(self, dev_ctrl: ChipDeviceCtrl, node_id: int, discriminator) -> CommissioningParameters:
+        try:
+            params = dev_ctrl.OpenCommissioningWindow(nodeid=node_id, timeout=900, iteration=1000,
+                                                      discriminator=discriminator, option=1)
+            return params
+
+        except Exception as e:
```

Comment:
You probably won't pass the linter with except Exception - this needs to be more specific. Try InteractionModelError here.

Reply:
I replaced expection and now I used InteractionModelError to handle the error.